x-pack/plugins/translations/translations/zh-CN.json

```diff
@@ -1552,7 +1552,6 @@
     "kbn.discover.fieldChooser.filter.hideMissingFieldsLabel": "隐藏缺失字段",
     "kbn.discover.fieldChooser.filter.indexAndFieldsSectionAriaLabel": "索引和字段",
     "kbn.discover.fieldChooser.filter.popularTitle": "常用",
-    "kbn.discover.fieldChooser.filter.resetFiltersButtonLabel": "重置筛选",
```

Comment:
Hm, just thinking about this... this is one thing I omitted in my original PR, mostly because Lens doesn't have. Should we add this option as well? Definitely not a blocker for this PR, just thinking it might be worth adding a backlog item if needed?

Reply:
A backlog item to discuss is a good idea! thx!